base/range.jl

```diff
@@ -711,7 +711,7 @@ julia> step(range(2.5, stop=10.9, length=85))
 """
 step(r::StepRange) = r.step
 step(r::AbstractUnitRange{T}) where {T} = oneunit(T) - zero(T)
```

Comment:
I wonder why this definition is used here? The docstring of `AbstractUnitRange` states that the step is `oneunit(T)`, which seems to differ from this definition.

Reply:
ask https://github.com/JuliaLang/julia/pull/27302